csrankings.csv

```diff
@@ -5040,6 +5044,7 @@ Heinz W. Schmidt,RMIT University,https://www.rmit.edu.au/contact/staff-contacts/
 Helen C. Purchase,University of Glasgow,http://www.dcs.gla.ac.uk/~hcp,4I7Q3nEAAAAJ
 Helen C. Shen,HKUST,http://repository.ust.hk/ir/AuthorProfile/shen-helen-c-m,W0Cx7ZAAAAAJ
 Helen Cameron,University of Manitoba,http://www.cs.umanitoba.ca/~hacamero,NOSCHOLARPAGE
+Helen M. Meng,Chinese University of Hong Kong,http://www.se.cuhk.edu.hk/people/hmmeng.html,NOSCHOLARPAGE

```

Comment:
This person (and some others added to CUHK) does not appear to be in the computer science department. 

http://www.cse.cuhk.edu.hk/en/people/faculty/by-profile

They are thus not eligible for inclusion; please remove them.

Reply:
Removed. Thanks for noticing!